tests/test-misc.c

```diff
@@ -479,71 +477,113 @@ test_pcap(void)
 	io = nmsg_io_init();
 	check_return(io != NULL);
 
-	phandle = pcap_open_offline("/tmp/http.cap", errbuf);
+	phandle = pcap_open_offline("./tests/private-tests/http_response.pcap", errbuf);
 	check_return(phandle != NULL);
 
 	pcap = nmsg_pcap_input_open(phandle);
 	check_return(pcap != NULL);
 
-	mod = nmsg_msgmod_lookup(NMSG_VENDOR_BASE_ID, NMSG_VENDOR_BASE_HTTP_ID);
+	mod = nmsg_msgmod_lookup(NMSG_VENDOR_BASE_ID, NMSG_VENDOR_BASE_PACKET_ID);
 	check_return(mod != NULL);
 
 	input = nmsg_input_open_pcap(pcap, mod);
 	check_return(input != NULL);
 
-	nmsg_pcap_input_set_raw(pcap, true);
+	nmsg_pcap_input_set_raw(pcap, false);
 
-#define BPF_FILTER_STRING	"tcp dst port 80 or tcp src port 80"
+	const char bpf_filter_string[] = "tcp dst port 80 or tcp src port 80";
 	struct timespec ts;
-	struct pcap_pkthdr *pphdr;
-	const uint8_t *pkdata;
+	struct nmsg_ipdg ni;
 
-	check_return(nmsg_pcap_input_setfilter_raw(pcap, BPF_FILTER_STRING) == nmsg_res_success);
-//	check_return(nmsg_pcap_input_setfilter(pcap, BPF_FILTER_STRING) == nmsg_res_success);
+	check_return(nmsg_pcap_input_setfilter_raw(pcap, bpf_filter_string) == nmsg_res_success);
+	check_return(nmsg_pcap_input_setfilter(pcap, bpf_filter_string) == nmsg_res_success);
```

Comment:
Let's keep this second one commented out because it's creating coverage illusion

Reply:
Fixed